src/devices/watts_thermostat.c

```diff
@@ -0,0 +1,199 @@
+/** @file
+    Watts WFHT-RF Thermostat
+
+    Copyright (C) 2022 Ådne Hovda <aadne@hovda.no>
+    based on protocol decoding by Christian W. Zuckschwerdt <zany@triq.net>
+    and Ådne Hovda <aadne@hovda.no>
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+*/
+
+#include "decoder.h"
+/**
+Watts WFHT-RF Basic Thermostat
+
+This code is based on a slightly older OEM system created by ADEV in France which
+later merged with Watts. The closest thing currently available seems to be
+https://wattswater.eu/catalog/regulation-and-control/radio-wfht-thermostats/electronic-room-thermostat-with-rf-control-wfht-rf-basic/,
+but it is not known whether they are protocol compatible.
+
+https://github.com/merbanan/rtl_433/issues/2230
```

Comment:
My inclination is not to have that sort of history link, and not to have analyzer output.  Rather, the info about how it works woudl be cleaned up and summarized and just presented.  I think we want to leave the messy path to understanding (not a criticism here -- it is *always* messy) behind.